Tests/SentryTests/SentryMsgPackSerializerTests.m

```diff
@@ -0,0 +1,88 @@
+#import "SentryMsgPackSerializer.h"
+#import <Foundation/Foundation.h>
+#import <XCTest/XCTest.h>
+
+@interface SentryMsgPackSerializerTests : XCTestCase
+
+@end
+
+@implementation SentryMsgPackSerializerTests
```

Comment:
No, I mean using some predefined dictionary with a couple of different values, serializing it with another message pack library, store the output to a file, add this file to our tests, and then comparing the results of the predefined dictionary with the stored file.

Reply:
Although the reading works in another message pack library (I tested it). The writing is different since I didn't optimise as explained in the SentryMsgPackSerializer.m line 35

```
        // We will always use the 4 bytes data length for simplicity.
        // Worst case we're losing 3 bytes.
```
As long we dont change the tests in this file that check for format, there is nothing to worry about.

Reply:
Then maybe add a snapshot. Take some sample data, serialize it with `SentryMsgPackSerializer`, validate the oupt against another message pack library, store the contents to a file, and use the file in the tests.